source/includes/configuration/_02-get-historical-snapshot.md.erb

```diff
@@ -22,26 +28,23 @@ $ curl 'https://ci.example.com/go/api/admin/config/5059cf548db9ea2d1b9192b45529c
 </cruise>
 ```
 
-Gets the configuration with the specified version.
+Gets the current configuration file.
 
 <%= available_since('14.3.0') %>
 
 ### HTTP Request
 
-`GET /go/api/admin/config/:commit_sha.xml`
+`GET /go/api/admin/config/config.xml`
 
 **Other convenience APIs**
 
-`GET /go/api/admin/config/config.xml`
+`GET /go/api/admin/config/current.xml`
 
     or
 
-`GET /go/api/admin/config/current.xml`
+`GET /go/api/admin/config/:md5.xml`
```

Comment:
> Should we drop this documentation? I think it's used by the ui currently to figure out conflicts.

This was a [documented API](https://www.go.cd/documentation/user/15.1.0/api/configuration_api.html), so we cannot yank it without providing a useful alternative.